### PR TITLE
Clean YouTube video gathering and added JS features

### PR DESCRIPTION
**Pull request by cjhbh3**
Major Updates:
Added Front-End Additions: Includes a button that currently fetches 5 hard-coded YouTube API results and displays them in iFrame spots on **index.html**. The front-end will be improved more, but this is a start to our design.

Tuning Updates:
- Within **YouTubeServlet.java**, added a parameter within the query of the YouTube API that specifies only videos. This was added to fix the issue of incidentally grabbing playlists.
- Additionally, commented out code relating to retrieving the video titles. This may be used in future uses, however currently it is not needed.

### CODE REVIEW

File: src/main/webapp/script.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
+var clicked = false;
+
+function getVideoResults() {
+  var i;
+  if (clicked == false) {
```
Comment (JabariD) [inline src/main/webapp/script.js:5]: nit: change to (!clicked)

File: src/main/webapp/script.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
+var clicked = false;
+
+function getVideoResults() {
```
Comment (JabariD) [inline src/main/webapp/script.js:3]: Up to you, I know this syntax looks a little verbose.

But you can modify this function to an async function just to make this a little bit more readable:
https://developers.google.com/web/fundamentals/primers/async-functions

File: src/main/webapp/script.js (hunk 1 above)
Comment (cjhbh3) [reply src/main/webapp/script.js:5]: noted

File: src/main/webapp/script.js (hunk 2 above)
Comment (cjhbh3) [reply src/main/webapp/script.js:3]: I'll look into it more.

File: src/main/java/com/google/sps/servlets/resources/YouTubeServlet.java [hunk 3]
```diff
@@ -52,41 +69,39 @@ public static YouTube getService() throws GeneralSecurityException, IOException
...
-        response.getWriter().println("Error: " + e);
-      }
+    public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {     
       try {
         YouTube youtubeService = getService();
-        long num = 1;
-        String templateLink = "www.youtube.com/watch?v=";
+        long results = 5;
+        // Use for creating links
+        String templateLink = "https://www.youtube.com/embed/"; //"https://www.youtube.com/watch?v=";
```
Comment (ptai7) [inline src/main/java/com/google/sps/servlets/resources/YouTubeServlet.java:77]: Remove unused comments.

**Pull request merged**